packages/@eris-boiler/common/src/utils/identity.ts

```diff
@@ -0,0 +1,5 @@
+export function identity <T> (
+  list: Array<T | null | undefined>,
+): Array<T> {
+  return list.filter((x) => x != null) as Array<T>
```

Comment:
eqeqeq?

Reply:
eh, I guess we could do the 2 eqeqeq, but kind of lazy

Reply:
linter explicitly allows this :)